src/raftstore/store/config.rs

```diff
@@ -102,6 +102,18 @@ impl Config {
                                 self.raft_log_gc_threshold));
         }
 
+        if self.region_max_size <= self.region_split_size {
+            return Err(box_err!("region max size {} must > split size {}",
+                                self.region_max_size,
+                                self.region_split_size));
+        }
+
+        if self.region_check_size_diff >= self.region_split_size {
```

Comment:
Yes, my mistake. What I actually wanted to say was `region_check_size_diff` can be larger than `region_split_size`.